src/variables/variable.ts

```diff
@@ -1,29 +1,29 @@
-
-
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
 
 export interface IVariable {
```

Comment:
Maybe we can consider making this interface based on [DebugProtocol.Variable](https://microsoft.github.io/debug-adapter-protocol/specification#Types_Variable)?

Reply:
I check   DebugProtocol.Variable and I think we can extend this interface or just replace and use DebugProtocol.Variable.